yaml/_type/pairs.ts

```diff
@@ -9,6 +9,8 @@ import type { Any } from "../_utils.ts";
 const _toString = Object.prototype.toString;
 
 function resolveYamlPairs(data: Any[][]): boolean {
+  if (data === null) return true;
```

Comment:
This check exists in the original (js-yaml). ref: https://github.com/nodeca/js-yaml/blob/0d3ca7a27b03a6c974790a30a89e456007d62976/lib/type/pairs.js#L8